mslib/utils/airdata.py

```diff
@@ -37,10 +37,10 @@
 import time
 
 import defusedxml.ElementTree as etree
-from mslib.msui.constants import MSUI_CONFIG_PATH
+from mslib.msui.constants import HOME
 
-
-OSDIR = fs.open_fs(MSUI_CONFIG_PATH).root_path
+DOWNLOAD_PATH = os.getenv("MSUI_CONFIG_PATH", os.path.join(HOME, ".config", "msui", "downloads", "aip"))
+OSDIR = fs.open_fs(DOWNLOAD_PATH).root_path
```

Comment:
when the downlads dir not exists, it crashes

`~/.config/msui$ rm -rf downloads/`

fs.errors.CreateFailed: root path '/home/reimar/.config/msui/downloads/aip' does not exist

Reply:
also you better do the join to the OSDIR.  
`OSDIR = fs.open_fs(MSUI_CONFIG_PATH).root_path`

 os.getenv("MSUI_CONFIG_PATH", ...
 
 means that when the env var is set, the second parameter isn't used.

Reply:
for example if I want to use my configurations from a different config dir
 
 export MSUI_CONFIG_PATH=~/tmp

I also want than on this dir a download folder and not what we have in 7.x

Reply:
> 

If I don't use the os.getenv function like this 
![image](https://user-images.githubusercontent.com/73490663/224819337-6ebf2b13-ebf0-4de3-a7ff-5a9bfb51fc78.png)

Then I get this error 
![image](https://user-images.githubusercontent.com/73490663/224820707-a5d97de3-4106-41c3-957d-02d4c7dae89f.png)

This error states that the variable [MSUI_CONFIG_PATH] has some values /tmp/tmp*
However when using setting the OSDIR value to the this(image given below)
![image](https://user-images.githubusercontent.com/73490663/224821912-26474920-cfb7-4c6f-b43d-491c0251498f.png)
The test successfully passes.

Reply:
> also you better do the join to the OSDIR. `OSDIR = fs.open_fs(MSUI_CONFIG_PATH).root_path`
> 
> os.getenv("MSUI_CONFIG_PATH", ...
> 
> means that when the env var is set, the second parameter isn't used.

Do you mean that "os.getenv" should not be used?